cfssl.tf

```diff
@@ -98,6 +76,27 @@ resource "aws_volume_attachment" "cfssl-data" {
   device_name = "/dev/xvdf"
   volume_id   = "${aws_ebs_volume.cfssl-data.*.id[count.index]}"
   instance_id = "${aws_instance.cfssl.*.id[count.index]}"
+
+  provisioner "remote-exec" {
+    when = "destroy"
+
+    inline = [
+      "sudo systemctl stop cfssl.service",
+      "sudo umount -d ${self.device_name}",
+      "sudo halt",
+    ]
+
+    connection {
+      bastion_host = "jumpbox.${var.environment}.uw.systems"
```

Comment:
We should rethink this in a bigger way. Afaik this is the first change that introduces a hard link to UW. Before this it *could* be a generic module.

Reply:
Ah, that's true. But the jumpboxes are really something that we use but cannot require it by anyone who wants to use this module. I somehow need to make these optional I guess